examples/dotnet/Examples/Unity.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MongoDB.Bson;
+using Realms;
+using Realms.Sync;
+using System.Threading.Tasks;
+
+public class Cube
+{
+
+    class Entity : RealmObject
+    {
+        [PrimaryKey]
+        [MapTo("_id")]
+        public ObjectId Id { get; set; } = ObjectId.GenerateNewId();
+
+        public int foo { get; set; }
+    }
+
+    private void Start()
+    {
+        // 1) CreateCopy();
+
+        // 2) Copy the Realm file created by above function into the Assets folder (path will be printed to the console).
+
+        // 3) ReadCopy();
+    }
+
+    private async void CreateCopy()
+    {
+        var app = App.Create("foo");
+        var user = await app.LogInAsync(Credentials.Anonymous());
+        var syncConfiguration = new SyncConfiguration("partition_key", user, "");
```

Comment:
```suggestion
        var syncConfiguration = new SyncConfiguration("partition_key", user);
```

If you leave the path empty you can just ignore all together, it's the same result.